src/app/component/eco-news/news-list/news-list.component.html

```diff
@@ -1,7 +1,7 @@
 <div>
   <div class="flex-container">
     <app-filter-news (gridOutput)="getFilterData($event)"></app-filter-news>
-    <a [routerLink]="'/create-news'">
+    <a [routerLink]="'/create-news'"  *ngIf="isLoggedIn">
```

Comment:
Please, write attributes in separate rows

Reply:
fix